packages/utils-tests-helpers/tests/hint-runner-helpers.ts

```diff
@@ -0,0 +1,39 @@
+import test from 'ava';
+
+import { positionToOffset } from '../src/hint-runner';
+
+/*
+ * ------------------------------------------------------------------------------
+ * positionToOffset tests
+ * ------------------------------------------------------------------------------
+ */
+
+test('positionToOffset - properly translates offset of file with LF line endings', (t) => {
+    const document = 'one\ntwo\nthree\nfour';
+
+    const actual = positionToOffset({column: 1, endColumn: 3, endLine: 2, line: 2}, document);
+    const expected = [9, 11];
+
+    t.true(actual[0] === expected[0] && actual[1] === expected[1], `Offset differs from expected.
+                Actual: [${actual[0]}, ${actual[1]}], Expected: [${expected[0]}, ${expected[1]}]`);
```

Comment:
Not blocking, but `t.deepEqual(actual, expected)` would also work here and provide the diff output for you.